ui-core/src/components/inputs/ComboBox/ComboBox.tsx

```diff
@@ -16,7 +16,13 @@ import cx from 'classnames';
 import useOutsideClick from '../../../hooks/useOutsideClick';
 import Input, { type InputProps } from '../Input';
 
-export type ComboBoxProps<T> = Omit<InputProps, 'value'> & {
+export type ComboBoxProps<T> = (
+  | (Omit<InputProps, 'value' | 'children' | 'required'> & {
+      required: boolean;
+      narrow?: undefined;
+    })
+  | (Omit<InputProps, 'value' | 'children' | 'narrow'> & { narrow: boolean; required?: undefined })
+) & {
```

Comment:
Why do we need to change this type? I think `InputProps` already has the correct `required` and `narrow` fields?